user_manager/signals.py

```diff
@@ -0,0 +1,34 @@
+"""
+Signals for User Manager app.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from .models import UserManagerRole
+
+
+def _upgrade_manager_role_entry(created, user):
+    """
+    Upgrade an email link to a proper FK link when a manager registers.
+    """
+
```

Comment:
Quality issue:
```
user_manager/signals.py:13 in private function `_upgrade_manager_role_entry`:
        D202: No blank lines allowed after function docstring (found 1)
```

Reply:
I found some of these docstring rules a bit too restrictive. For instance the need to have a period at the end of the first line of the docstring. 

Also, do you know how we could configure this to run all tests even if earlier ones failed? I actually prefer that behaviour since it means you get a more useful list of things to fix.

Reply:
I prefer that too but I don't know off hand.